app/models/manageiq/providers/amazon/cloud_manager.rb

```diff
@@ -103,11 +103,7 @@ def self.hostname_required?
   end
 
   def self.default_blacklisted_event_names
-    %w(
-      ConfigurationSnapshotDeliveryCompleted
-      ConfigurationSnapshotDeliveryStarted
-      ConfigurationSnapshotDeliveryFailed
-    )
+    Settings.blacklisted_event_names
```

Comment:
When the event names are at `:amazon:` this will of course change.
I also see in the core repo `::Settings`.

Also, how about making this change in the base class?
Or do you want to prove it first here?